### ISSUE TITLE
[Accessibility] keyboard only users cannot see tooltips

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#120210]
- VS Code Version: 1.54.3 stable 
- OS Version: Windows

Steps to Reproduce:

1. go to  vs code explorer 
2. navigate to icons via "tab" key ( icons in "open editors", icons in "outline" etc.)
3. Key board only user should be able to see tooltip 
**Actual :**  keyboard only user cannot see tooltip , users only see tooltips when they hover over via mouse. 

Does this issue occur when all extensions are disabled?: Yes

**Comment by setaskin** [comment]
Hi @isidorn , we have a bug assigned to us for this problem, I see same behavior on vscode, I though root cause might be coming from vscode repo

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/52016
